website_product_share/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+============
+Allow users to enable/disable share button on each product page
```

Comment:
This modules brings the users the possibility to globally enable/disable a share button that will be displayed in each product page.